services/hapi/src/libs/sync-ratings.js

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env node
+const { JsonRpc } = require('eosjs')
+const fetch = require('node-fetch')
+const massive = require('massive')
+
+const dbConfig = require('../config/dbConfig')
+
+const EOS_API_ENDPOINT = process.env.EOS_API_ENDPOINT || 'https://jungle.eosio.cr'
+
+// gets data from blockchain
+const getUserRatings = async () => {
+  const eos = new JsonRpc(EOS_API_ENDPOINT, { fetch })
+
+  let ratings = await eos.get_table_rows({
+    json: true,
+    code: 'rateproducer',
+    scope: 'rateproducer',
+    table: 'ratings',
+    limit: 1000,
+    reverse: false,
+    show_payer: false
+  })
+  //console.log(ratings)
+  return ratings
+}
+
+// updates the postgresdb
```

Comment:
this script is meant to sync the blockchain tables with postgres. 

If the DB is ever deleted we can inport the ratings using this script.